x-pack/plugins/security_solution/server/lib/telemetry/routes/index.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import type { Logger, IRouter } from '@kbn/core/server';
+import type { ITelemetryReceiver } from '../receiver';
+import type { ITelemetryEventsSender } from '../sender';
+import { TaskMetricsService } from '../task_metrics';
+import { createTelemetryIndicesMetadataTaskConfig } from '../tasks/indices.metadata';
+
+// TODO: just to test the POC, remove
+export const getTriggerIndicesMetadataTaskRoute = (
+  router: IRouter,
+  logger: Logger,
+  receiver: ITelemetryReceiver,
+  sender: ITelemetryEventsSender
+) => {
+  router.get(
+    {
+      path: '/internal/trigger-indices-metadata-task',
+      validate: {
+        query: schema.object({
+          maxPrefixes: schema.maybe(schema.number()),
+          maxGroupSize: schema.maybe(schema.number()),
+        }),
+      },
+    },
+    async (_context, request, response) => {
+      const taskMetricsService = new TaskMetricsService(logger, sender);
+      const task = createTelemetryIndicesMetadataTaskConfig();
+      const timeStart = performance.now();
+
+      const { maxPrefixes, maxGroupSize } = request.query;
+
+      logger.info(
+        `Triggering indices metadata task with pageSize: ${maxPrefixes} and dataStreamsLimit: ${maxGroupSize}`
+      );
+
+      let msgSuffix = '';
```

Comment:
I don't think you want GC calls, and memory usage stats here.  Forcing GCs is almost always wrong, and could negatively affect performance.  The memory numbers will be completely useless, as there will be other tasks running, using memory.  We don't know of any way to get per-task memory usage numbers like this.

Reply:
Yes, you are right. But this route is just for testing purposes and won't be merged